### PR TITLE
Media query added to design desktop section

### PR DESCRIPTION
**Pull request by abiodunraheem**
## Media query added to design desktop section

- I  parse a Figma design to create a UI.
- I created UIs adaptable to different screen sizes using media queries.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -50,126 +50,424 @@ <h1>Hey There.
...
+                    </ul>
+                  </li>
+                </ul>
+              </div>
+              <div class="projectButton">
+                <button type="button">See Project</button>
+              </div>
+            </div> 
+          </section>
+       </main>
```
Comment (CollinsTatang) [inline index.html]: - [ ] Please check these lines of code only one item display and try to follow the figmal design 
### **Take a Look **
your-car
### **Check figmal Design**
figma-cart

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @abiodunraheem, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] Please fix your social media icons
### **Take a Look**
your-social

### **See the figmal design**
figmal-social

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -50,126 +50,424 @@ <h1>Hey There.
...
+                    <ul>
+                      <li>html</li>
+                    </ul>
+                  </li>
+                  <li class="projectBootstrap">
+                    <ul>
+                      <li>bootstrap</li>
+                    </ul>
+                  </li>
+                  <li class="projectRuby">
```
Comment (MeqdamAlqudah) [inline index.html:167]: - [ ] Please make sure that all the projects are appearing on the screen and make sure they looks exactly like Figma design 
image

image

I think the problem here comes from the quires make sure that you are using them correctly 
image

File: index.html [hunk 3]
```diff
@@ -50,126 +50,424 @@ <h1>Hey There.
...
-                    <ul class="grid-container">
-                        <li class="grid-item1"><a href="#">css</a></li>
-                        <li class="grid-item2"><a href="#">html</a></li>
-                        <li class="grid-item3"><a href="#">bootstrap</a></li>
-                        <li class="grid-item4"><a href="#">Ruby</a></li>
-                    </ul>
-                    <button name="button" type="submit" class="btn-1">See Project</button>
-                </div>
+          <div class="recent-work">
+            <div class="image-1">
```
Comment (MeqdamAlqudah) [inline index.html:59]: - [ ] Please make sure that your mobile version still work even after I change back to it 
I think this problem comes from media queries used poorly, also make sure to 
image

File: index.html [hunk 4]
```diff
@@ -50,126 +50,424 @@ <h1>Hey There.
...
+                A daily selection of privately personalized reads;
+                no accounts or sign-ups required. 
+                has been the industry's standard dummy text ever since the 1500s, 
+                when an unknown printer took a standard dummy.
+              </p>                    
+              <ul class="grid-container">
+                <li class="grid-item1"><a href="#">css</a></li>
+                <li class="grid-item2"><a href="#">html</a></li>
+                <li class="grid-item3"><a href="#">bootstrap</a></li>
+                <li class="grid-item4"><a href="#">Ruby</a></li>
```
Comment (MeqdamAlqudah) [inline index.html:74]: - [ ] Please make sure to align the navigation links exactly as Figma design i.e fix the distance between the links in Figma design there is more margin between the links

image

image

**Review by MeqdamAlqudah — COMMENTED** [review]
Hi @abiodunraheem , 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Pull request closed without merging**